### ISSUE TITLE
Randomly throws ReadTimeOutException

### ISSUE THREAD

**Issue by PrexorJustin** [Prokyo/ProkyoNet#69]
After a certain amount of time either the client or the server throws a ReadTimeOutException, maybe we need to send KeepAlive packets.

**Comment by Masy** [comment]
Yes, thats due to nettys automatic timeout after 30 seconds i think. Currently the user has to do the heartbeat by himself, the packet is already registered as a default packet though https://github.com/Prokyo/ProkyoNet/blob/0d0752825dbfcdc91570c1078438a79d9d912139/common/src/main/java/de/prokyo/network/common/packet/PacketRegistry.java#L23
